src/RA_Defs.h

```diff
@@ -89,15 +90,31 @@
 #define _SILENCE_CXX17_ITERATOR_BASE_CLASS_DEPRECATION_WARNING
 
 //	RA-Only
-#include "rapidjson/include/rapidjson/document.h"
-#include "rapidjson/include/rapidjson/writer.h"
-#include "rapidjson/include/rapidjson/filestream.h"
-#include "rapidjson/include/rapidjson/error/en.h"
+#define RAPIDJSON_HAS_STDSTRING 1
+#pragma warning(push, 1)
+// This is not needed the most recent version
+#define _SILENCE_ALL_CXX17_DEPRECATION_WARNINGS
+//	RA-Only
+#include <rapidjson/document.h> // has reader.h
+#include <rapidjson/writer.h> // has stringbuffer.h
+#include <rapidjson/filestream.h>
+#include <rapidjson/stringbuffer.h>
+#include <rapidjson/error/en.h>
+#pragma warning(pop)
+
 using namespace rapidjson;
 extern GetParseErrorFunc GetJSONParseErrorStr;
 #pragma warning(pop)
+
+
+using namespace std::string_literals;
+//using namespace std::chrono_literals; we could use this later
+namespace ra {}
+using namespace ra;
```

Comment:
what does this do (declaring an empty namespace and using it)?

Reply:
also, per CONTRIBUTING, you can use things in a namespace, but not the entire namespace.

Reply:
These will be gone.